supernode/examples/configs/mainnet.yml

```diff
@@ -1,11 +1,28 @@
+# please provide pastel configs in pastel.conf
+log-level: info
+log-file:  # e.g: logs.json - by default, logs are only written to stdout
+quiet: false
+temp-dir:
+work-dir:
+rq-files-dir: 
+dd-service-dir:
```

Comment:
`dd-service-dir` should be set to `~/pastel_dupe_detection_service`

Reply:
@nguyen202  defaults are already set in code.